libs/db-browser/src/main/java/com/oceanbase/tools/dbbrowser/parser/PLParser.java

```diff
@@ -46,11 +50,38 @@
 @Slf4j
 public class PLParser {
 
+    private final static Cache<String, CacheElement<ParseMysqlPLResult>> OB_MYSQL_PARSE_CACHE =
+            Caffeine.newBuilder()
+                    .maximumSize(1000)
+                    .expireAfterWrite(10, TimeUnit.MINUTES)
+                    .build();
+    private final static Cache<String, CacheElement<ParseOraclePLResult>> OB_ORACLE_PARSE_CACHE =
+            Caffeine.newBuilder()
+                    .maximumSize(1000)
+                    .expireAfterWrite(10, TimeUnit.MINUTES)
+                    .build();
+    private final static Cache<String, CacheElement<ParseOraclePLResult>> ORACLE_PARSE_CACHE =
```

Comment:
because ob-oracle and oracle use different parser and will get different result. we'd better not use the result that ob-oracle's parser gives stands for the result that oracle' s parser gives.